draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1184,7 +1192,7 @@ According to the definition of the MP_FAST_CLOSE option in {{MP_FAST_CLOSE}}, a
 
 * setting handover or aggregation mode
 
-* MP-specific congestion mechanisms
+* MP-specific congestion mechanisms (e.g., to support multiple congestion mechanisms for multiple paths?) 
```

Comment:
Probably that is an historic leftover. If we agree on subflow individual CC, also DCCP's Feature Negotiation can be used.